README.md

```diff
@@ -155,11 +161,42 @@ The following functions differ from the `CentralizedBlockRelay`:
     - *_tallyMerkleRoot*: the root hash of the tallies-only merkle tree as contained in the block header.
     - *_previousVote*: the previousVote that this posted block's vote extends.
 
+
 ## Known limitations:
 
 - `CentralizedBlockRelay`: as the name suggests, this block relay is centralized, only the deployer of the contract is able to push blocks.
 - `ActiveBridgeSetBlockRelay`: the ABS stays the same until a block is finalized. A block can be proposed more than once by the same ABS member and only for one epoch previous to the current epoch.
 
+
+## Benchmark
```

Comment:
I dont know why bu thte benchmark looks like this

![image](https://user-images.githubusercontent.com/52961116/108057305-c1717700-7052-11eb-8431-c2b9c4477b79.png)

Reply:
I will review

Reply:
Should be fixed now! :)